### PR TITLE
Logging capabilities of apps

### PR DESCRIPTION
**Pull request by iamhks**
closes odk-x/tool-suite-X#81

#### What is included in this PR?
Documented the logging capabilities of apps.

Hi @wbrunette , these are my initial improvements. Please review them and tell me what all changes need to be done.

### CODE REVIEW

File: odkx-src/logging-capabilities-of-apps.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Logging Capabilities of Apps
+=================================
+There are three types of logging that exists.
```
Comment (elmps2018) [inline odkx-src/logging-capabilities-of-apps.rst:3]: exist, singular.

File: odkx-src/logging-capabilities-of-apps.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+ODK-X Logging System
+----------------------
+
+The ODK-X log files are stored in the directory :file:`/sdcard/opendatakit/default/output/logging` of your Android device. If you do not have an external storage, the same directory would start with Internal Storage instead of sdcard.  If you would like to zip all of your ODK-X data and share it please watch `this video <https://www.youtube.com/watch?v=jzqu7w4VepM>`_ . 
+
+Android Logging
+----------------------
+
+The Android logging system provides a mechanism for collecting and viewing system debug output. Logs from various applications and portions of the system are collected in a series of circular buffers, which then can be viewed and filtered by the `logcat` command. You can use `logcat` from an Android Debug Bridge shell to view the log messages. You can also see it Android Studio.
```
Comment (elmps2018) [inline odkx-src/logging-capabilities-of-apps.rst:13]: see it in Android Studio (in missing)

File: odkx-src/logging-capabilities-of-apps.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Sync-Endpoint Docker
+----------------------
+
+To fetch logs of a container, you can run the command
+
+.. code-block:: console
+
+  $ docker logs [OPTIONS] CONTAINER
+
+For instance, you would like to fetch logs of ODK-X docs. The command would be
```
Comment (elmps2018) [inline odkx-src/logging-capabilities-of-apps.rst:31]: does it make sense to use an example of docs on sync endpoint?

**Review by elmps2018 — COMMENTED** [review]
Great to see this getting documented! A few minor edits.

**Comment by iamhks** [comment]
Thankyou @elmps2018 I have made the required changes.

File: odkx-src/logging-capabilities-of-apps.rst [hunk 4]
```diff
@@ -10,7 +10,7 @@ The ODK-X log files are stored in the directory :file:`/sdcard/opendatakit/defau
 Android Logging
 ----------------------
 
-The Android logging system provides a mechanism for collecting and viewing system debug output. Logs from various applications and portions of the system are collected in a series of circular buffers, which then can be viewed and filtered by the `logcat` command. You can use `logcat` from an Android Debug Bridge shell to view the log messages. You can also see it Android Studio.
+The Android logging system provides a mechanism for collecting and viewing system debug output. Logs from various applications and portions of the system are collected in a series of circular buffers, which then can be viewed and filtered by the `logcat` command. You can use `logcat` from an Android Debug Bridge shell to view the log messages. You can also see it Android Studio (in missing).
```
Comment (elmps2018) [inline odkx-src/logging-capabilities-of-apps.rst:13]: Not quite sure what "in missing" means here?

**Review by elmps2018 — COMMENTED** [review]
Thanks I think that addressed all my edits! @wbrunette or @linl33 should check that this all makes technical sense.

File: odkx-src/logging-capabilities-of-apps.rst (hunk 4 above)
Comment (iamhks) [reply odkx-src/logging-capabilities-of-apps.rst]: Oops! that was lame😅 corrected it!

**Comment by elmps2018** [comment]
This looks good to me now! Hoping @linl33 or @wbrunette who know the content better can also take a look!

**Review by wbrunette — APPROVED** [review]

**Pull request merged**